lib/shoryuken/manager.rb

```diff
@@ -13,22 +12,22 @@ def initialize(fetcher, polling_strategy)
       @queues = Shoryuken.queues.dup.uniq
 
       @done = Concurrent::AtomicBoolean.new(false)
-      @dispatching = Concurrent::AtomicBoolean.new(false)
 
       @fetcher = fetcher
       @polling_strategy = polling_strategy
 
-      @heartbeat = Concurrent::TimerTask.new(run_now: true,
```

Comment:
Not sure if the `dispatch_async` in `processor_done` does anything given the normal dispatcher re-runs at the end anyway.

But the ensure and re-run should keep the dispatch loop running I think. 👍